frontend/src/app/containers/ExperimentPage.js

```diff
@@ -383,7 +383,19 @@ export class ExperimentDetail extends React.Component {
                         {measurements && <section
                               className={classnames('measurements', { highlight: highlightMeasurementPanel })}>
                           <h3 data-l10n-id="measurements">Your privacy</h3>
-                          <div data-l10n-id={this.l10nId('measurements')} className="measurement" dangerouslySetInnerHTML={createMarkup(measurements)}></div>
+                          <div data-hook="measurements-html" className="measurement">
```

Comment:
🗡  (that was me typing a grinning face, but it autocompleted to a dagger so gonna go with that)